core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service.ts

```diff
@@ -28,13 +28,16 @@ import { ExplorationEditorPageConstants } from
 })
 export class TranslationTabActiveModeService {
   private activeMode: string = null;
+  private _updateTranslationActiveModeEventEmitter = new EventEmitter<string>();
```

Comment:
TBH I don't understand, from the name, what this is doing.

If it's meant to emit a signal when the activeMode is changed (whether to translation or voiceover), then perhaps call it _activeModeChangedEventEmitter ?

Also adding @srijanreddy98 to verify that the usage of the event emitter here makes sense within our planned frontend structure.

Reply:
Usage is almost in line with what we have at the moment. The naming could be better. `_activeModeChangedEventEmitter` suggested by Sean seems well suited for this.